src/components/Cards/CourseCard.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react'
+import { Text, View, StyleSheet } from 'react-native'
+
+class CourseCard extends Component {
+  styles = StyleSheet.create({
+    titleText: {
+      fontSize: 20,
+      fontWeight: 'bold',
+    },
+
+    card: {
+      backgroundColor: 'rgba(0,0,0,0.2)',
+      width: '50%',
+    },
+    container: {
+      padding: '0.01em 16px',
+    },
+    blue: {
+      color: 'white',
+      backgroundColor: '#2196F3',
+    },
+  })
+
+  constructor(props) {
```

Comment:
We probably shouldn't pass in the whole entire course info object as a single prop. It makes to code a bit ambiguous and we don't use all of the data anyway. Maybe we should pass in `courseTitle`, `courseDescription`, and `courseId` as seperate props

Reply:
fixed